extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -255,15 +258,16 @@ export abstract class AzureAuth implements vscode.Disposable {
 	}
 
 	public async getToken(tenant: Tenant, resource: Resource, postData: AuthorizationCodePostData | TokenPostData | RefreshTokenPostData): Promise<OAuthTokenResponse> {
+		Logger.write(LogLevel.Verbose, 'Fetching token');
 		const tokenUrl = `${this.loginEndpointUrl}${tenant.id}/oauth2/token`;
 		const response = await this.makePostRequest(tokenUrl, postData);
-
+		Logger.write(LogLevel.Verbose, `Token: ${response}`);
```

Comment:
I'm not sure logging the actual token here is useful at all - and that's pretty dangerous to log to a file on disk. Is there specific information here you're looking to gather?

If you do end up thinking it'd be useful you should definitely be using the pii function to ensure it's logged only when the user allows it.